mobbex/Models/Config.php

```diff
@@ -48,7 +48,7 @@ public function getSettings($key = 'key')
         $settings = [];
 
         foreach ($this->getConfigForm()['form']['input'] as $input)
-            $settings[$input[$key]]  = !\Configuration::get($input['name']) ? $input['default'] : \Configuration::get($input['name']);
+            $settings[$input[$key]]  = \Configuration::getIdByName($input['name']) ? $input['default'] : \Configuration::get($input['name']);
```

Comment:
Acá faltó agregar el not antes del getIdByName, hace el efecto contrario al deseado